.github/workflows/python-tox.yml

```diff
@@ -11,41 +11,104 @@ jobs:
       matrix:
         tox_env:
           - py37-django32-wagtail215
+          - py37-django32-wagtail216
```

Comment:
I wonder if it would be best to adopt a matrix here. This long list is open to mistakes.

See this as an example: https://github.com/torchbox/wagtail-wordpress-import/blob/9b07de5e0d3996a9f0d6b6406c9728db4a150021/.github/workflows/test.yml#L17 and then you could include the exclude options. The list will then be a lot shorter.

The example I gave started life from: https://github.com/wagtail/cookiecutter-wagtail-package

Reply:
I used include instead here:
https://github.com/labd/wagtail-2fa/blob/a522c3c3dbc044b54d57a4dd96e12169033e931f/.github/workflows/python-tox.yml

And updated the tox.ini to match:
https://github.com/labd/wagtail-2fa/blob/a522c3c3dbc044b54d57a4dd96e12169033e931f/tox.ini